apps/web/src/pages/index.tsx

```diff
@@ -1,23 +1,71 @@
 import { Layout } from "../helper/layout";
 import type { ReactElement } from "react";
+import Image from "next/image";
+import Link from "next/link";
 
 export default function Landing() {
   return (
-    <>
+    <div className="flex flex-col h-[calc(100vh-6rem)] justify-between content-start gap-9">
       <div>
-        <h1 className="text-9xl font-sora">
-          <span>SK</span>
-          <span className="text-techdev-gradient-radial">TechDev</span>
-        </h1>
-        {/* <Button variant="contained" color="primary">
-          Join us
-        </Button> */}
+        <div>
+          <h1 className="text-9xl font-sora">
+            <span>SK</span>
+            <span className="text-techdev-gradient-radial">TechDev</span>
```

Comment:
There should be line break opportunities here. This currently doesn't break properly on mobile.